luigi/task.py

```diff
@@ -440,10 +440,12 @@ def _requires(self):
         provide consistent end-user experience), yet need to introduce
         (non-input) dependencies.
 
-        Must return an iterable which among others contains the _requires() of
-        the superclass.
+        Must return a structure of Tasks which among others contains
+        the _requires() of the superclass. The return value can be a single
+        Task, a list of Task instances, or a dict whose values are
+        Task instances.
         """
-        return flatten(self.requires())  # base impl
+        return self.requires()  # base impl
```

Comment:
If someone wants the `flatten`ed version of requirements, they should use `deps()`.